holoviews/plotting/bokeh/__init__.py

```diff
@@ -116,11 +118,11 @@
 
 # Charts
 options.Curve = Options('style', color=Cycle(), line_width=2)
-options.Scatter = Options('style', color=Cycle())
+options.Scatter = Options('style', color=Cycle(), size=np.sqrt(6))
 options.ErrorBars = Options('style', color='black')
 options.Spread = Options('style', fill_color=Cycle(), fill_alpha=0.6, line_color='black')
 options.Histogram = Options('style', fill_color="#036564", line_color="#033649")
-options.Points = Options('style', color=Cycle())
+options.Points = Options('style', color=Cycle(), size=np.sqrt(6))
```

Comment:
True, will add it. Point sizes in bokeh are widths while matplotlib uses area and has a default of 6.